view/rooms/app.js

```diff
@@ -75,6 +77,89 @@ function createHeader(bot) {
   return section;
 }
 
+/**
+ * Setup the socket.io client to listen to a namespace, and once sockets
+ * are connected install the bots in the room.
+ *
+ * @param  {Array} bots - Array of Bots
+ */
+function setupSocketIOClient(bots) {
+  bots.forEach((bot) => {
+    botInfo[bot.body.id] = bot.body.name;
+  });
+
+  const socket = _io('/', { transports: ['websocket'] });
+
+  const settingsChangedEventName =
+    'refocus.internal.realtime.room.settingsChanged';
+  const botActionsUpdate =
+    'refocus.internal.realtime.bot.action.update';
+  const botDataAdd =
+    'refocus.internal.realtime.bot.data.add';
+  const botDataUpdate =
+    'refocus.internal.realtime.bot.data.update';
+  const botEventAdd =
+    'refocus.internal.realtime.bot.event.add';
+
+  socket.on('connect', () => {
+    console.log('Socket Connected');
```

Comment:
Did you mean to leave console.log on? If you want browser-side debug/logging, please consider putting it behind a flag like we do with the perspectives page (?debug=REALTIME)